### ISSUE TITLE
Automatically update examples

### ISSUE THREAD

**Issue by nichollsh** [FormingWorlds/PROTEUS#165]
It is useful to store example results for several reasons:
 - when demonstrating what the model can do
 - knowing what sort of thing to expect when running the model
 - comparison with previous results following changes to the model
 - sanity checks

There are currently two examples, which I ran manually from the 'default' and 'dummy' configs, and then copied to the examples folder with 'tools/MakeExample.sh`. However, they are now out of date.

It would be better if these examples were automatically updated. This could potentially operate via a GitHub action, triggered when a release is created.

**Comment by nichollsh** [comment]
A prerequisite for this is to have PROTEUS running via GitHub actions. This should be done through a separate issue, and is also related to implementing some more tests.

**Comment by stefsmeets** [comment]
Can't do this on the main branch because of branch protection.

**Comment by nichollsh** [comment]
As in, because we require changes to be made through a PR?

**Comment by stefsmeets** [comment]
Yeah, exactly

**Comment by nichollsh** [comment]
Makes sense. Maybe in we could consider having the model run examples and package them as an artefact? Particularly if the examples are only to be run when a release is generated. Would this have permissions issues?

**Comment by stefsmeets** [comment]
I'm not entirely sure how that would work, artifacts are not great to work with (they are more of a developer thing) and you cannot link them directly, which is a major bottleneck.
